pybop/optimisers/base_optimiser.py

```diff
@@ -109,9 +112,19 @@ def set_base_options(self):
         """
         Update the base optimiser options and remove them from the options dictionary.
         """
-        self.x0 = self.unset_options.pop("x0", self.x0)
-        self.bounds = self.unset_options.pop("bounds", self.bounds)
-        self.sigma0 = self.unset_options.pop("sigma0", self.sigma0)
+        # Set initial values
```

Comment:
A comment to go alongside the below changes would be helpful. At first glance it appears self.x0 is being set to None if a kwarg isn't applied. 

```suggestion
        # Set initial values, if x0 is None, initial values are unmodified.
```